src/Radarr.Api.V3/swagger.json

```diff
@@ -3883,13 +3883,18 @@
         "properties": {
           "coverType": {
             "type": "string",
-            "enum": [
-              "poster",
-              "fanart"
-            ]
+            "enum": ["poster", "fanart"],
+            "example": "poster"
           },
           "url": {
-            "type": "string"
+            "type": "string",
+            "summary": "A path that can be used together with the host to find the image - requires API key",
```

Comment:
As this isn't an endpoint and is part of another object, i dont think ` - requires API key` is necessary? I would also mention that this path is in relation to the `AppData directory` found in the UI System screen since that path is not always the same.

`Image location relative to the AppData directory setting found under System in the UI` or something like that maybe?

Reply:
My intention was to convey that one can't just request the path together with the host. You have to add the apikey parameter for it to actually function. This wasn't super clear to me initially hence why I added this.

I'm however entirely open to change it to your suggestion and can do so later today/tomorrow. :)

Reply:
Guess i dont follow. I dont need an API key to access an image on disk (that is how i am understanding the response)

`C:\ProgramData\Radarr\MediaCover\2\poster.jpg` works fine & if you mean to query just the image, it is part of the movie object endpoint which already has error codes and mentions the api key requirement. There shouldn't need to be a note on any parameter that it cant be accessed without an api key since the endpoint cant be. Again, maybe i am just not understanding though.

Reply:
Q cleared it up for me, as I was making an assumption I shouldn't have.

Reply:
When accessing from disc that's correct. But when you request it via the API you need an API key to see it.

It seems as if we have a miscommunication. I believe the `url` returned is a path that can be used via the API only. It doesn't actually include the full hard drive path. Though I didn't actually try this. I did try to use this path via the domain and for that I required an API key. I guess it might be both?

Reply:
So maybe we can go with: `Relative image location. Can either be used together with the AppData directory setting for a full local path or be used to request via HTTP together with the host and apikey query parameter`

A little lengthy but complete, otherwise we can maybe just describe it as the relative image location and leave it at that.

What do you think?

Reply:
I think your original is likely fine and me just making an incorrect assumption of usecase and what was trying to be explained.